app/components/user-dropdown.js

```diff
@@ -3,8 +3,6 @@ import Ember from 'ember';
 export default Ember.Component.extend({
   classNames: ['user-dropdown'],
 
-  session: Ember.inject.service(),
```

Comment:
I think that's actually something ember wants to walk away from and the preferred approach is to explicitly inject services this way, not through initializers.